lib/ljglibs/gdk/pixbuf.moon

```diff
@@ -0,0 +1,20 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+ffi = require 'ffi'
+glib = require 'ljglibs.glib'
+core = require 'ljglibs.core'
+require 'ljglibs.cdefs.gdk'
+
+C = ffi.C
+
+core.define 'GdkPixbuf', {
+  get_from_window: (window, x, y, width, height) ->
+    pixbuf = C.gdk_pixbuf_get_from_window(window, x, y, width, height)
+    error 'Failed to get pixbuf' unless pixbuf
+    ffi.gc(pixbuf, C.g_object_unref)
+    pixbuf
+
+  save: (filename, type) =>
```

Comment:
In this case I think it perhaps doesn't make sense to adhere slavishly to the API with regards to the optional parameters for `save` and `savev`, but it would be nice to make this complete in the sense that it would accept as its last parameter a Lua table with optional parameters that we'd pass into `savev`. There's a wrapper helper in the glib module, `char_p_arr`, that takes a table and produces a NULL terminated array suitable for this. So, given a last parameter `opts = {}`, something like this:

```moonscript
option_keys = glib.char_p_arr [k for k in pairs opts]
option_values = glib.char_p_arr [v for k, v in pairs opts]
```

And pass those to `savev`.